Makefile

```
@@ -52,3 +49,13 @@ setup-nunet-dms:
 reset-nunet-dms:
 	@bash -x src/scripts/reset-nunet-dms.sh
 
+bazel:
+	@bash ./src/scripts/bazel.sh $(filter-out $@, $(MAKECMDGOALS))
+
+test-all: build-image
+	$(MAKE) bazel test //...
```

The suggested change below is not necessary if we figure out how to set the same behavior in `.bazelrc`:
```suggestion
	$(MAKE) bazel test --test_output=errors --build_tests_only --test_tag_filters=-skip" //...
```

---

See [here](https://github.com/singnet/das/pull/267#discussion_r1958062364).